docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: '3.7'
```

Comment:
If this file is supposed to be in the repository, can a clarification be added to `README.md` describing a proper way to use it?

Reply:
Ran some additional tests to validate removing this won't be an issue. I will remove it.